<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeleniumExport</class>
 <widget class="QDialog" name="SeleniumExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="testCaseLabel">
       <property name="text">
        <string>Test Case</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="testCaseComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="browserLabel">
       <property name="text">
        <string>Browser</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="browserComboBox">
       <item>
        <property name="text">
         <string>Firefox</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Chrome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Opera</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Safari</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Internet Explorer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Android</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>iPhone</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>iPad</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="systemLabel">
       <property name="text">
        <string>System</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="systemComboBox">
       <item>
        <property name="text">
         <string>Linux</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Windows</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Android</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mac</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="selectPathButton">
       <property name="text">
        <string>Select path</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="pathLabel">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="pathLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="boxButton">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boxButton</sender>
   <signal>accepted()</signal>
   <receiver>SeleniumExport</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boxButton</sender>
   <signal>rejected()</signal>
   <receiver>SeleniumExport</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
